pkg/login/interactive_login.go

```diff
@@ -46,6 +56,52 @@ func InteractiveLogin(config *config.Config) error {
 	return nil
 }
 
+// DisplayName returns the display name for a successfully authenticated user
+func DisplayName(account *Account, baseURL string, apiKey string) (string, error) {
+	// Account will be nil if user did interactive login
+	if account == nil {
+		acc, err := getAccount(baseURL, apiKey)
+		if err != nil {
+			return "", err
+		}
+
+		account = acc
+	}
+	displayName := account.Settings.Dashboard.DisplayName
+
+	return displayName, nil
+}
+
+// helper method to get the account for DisplayName
+func getAccount(baseURL string, apiKey string) (*Account, error) {
```

Comment:
As you mentioned, this function is duplicated from [`login_message.go`](https://github.com/stripe/stripe-cli/blob/master/pkg/login/login_message.go#L65-L92). Let's find a good location for this function and have it shared between the files using it.

Reply:
Agreed! I think that since it resides in the `login` logic, we can create a separate file in the `login` folder and have `login_message.go` as well as `interactive_login.go` share these functions